### PR TITLE
feat: Add legal notice and privacy policy links to help page

### PR DESCRIPTION
**Pull request by Pytal**
## Summary

image

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

File: apps/settings/lib/Controller/HelpController.php [hunk 1]
```diff
@@ -107,6 +113,9 @@ public function help(string $mode = 'user'): TemplateResponse {
 			$urlAdminDocs = $this->urlGenerator->linkToDocs('admin');
 		}
 
+		$legalNoticeUrl = $this->appConfig->getValueString('theming', 'imprintUrl');
+		$privacyUrl = $this->appConfig->getValueString('theming', 'privacyUrl');
```
Comment (susnux) [inline apps/settings/lib/Controller/HelpController.php:117]: Maybe better inject the `ThemingDefaults` and use the getters?

Comment (Pytal) [reply apps/settings/lib/Controller/HelpController.php:117]: Thought about that but even though theming is always enabled it is still under OCA namespace

**Review by susnux — APPROVED** [review]

**Review by sorbaugh — APPROVED** [review]

**Review by jancborchardt — COMMENTED** [review]
Thank you! :)

**Comment by AndyScherzinger** [comment]
@Pytal I added a backport since 30 is quite far away, so having it in at least for 29 would be great 👍

**Pull request merged**